source/blog/2018-06-29-ember-3-2-released.md

```diff
@@ -0,0 +1,295 @@
+---
+title: Ember 3.2 Released
+author: Robert Jackson, Melanie Sumner & Kenneth Larsen
+tags: Releases, 2018, 3, 3.2
+responsive: true
+---
+
+Today the Ember project is releasing version 3.2.0 of Ember.js, Ember Data, and Ember CLI.
+
+This release kicks off the 3.3 beta cycle for all sub-projects. We encourage our
```

Comment:
@MelSumner this still mentions the beta

Reply:
@Turbo87 yes- I thought since that is still accurate (the beta cycle is now kicked off) it would be okay to leave it in there.